lc-hax/Scripts/Modules/Possession/PossessionMod.cs

```diff
@@ -171,16 +211,26 @@ internal sealed class PossessionMod : MonoBehaviour {
         enemy.transform.position = characterMovement.transform.position;
     }
 
-    // Disables/enables colliders of the possessed enemy
-    void SetEnemyColliders(EnemyAI enemy, bool enabled) =>
-        enemy.GetComponentsInChildren<Collider>().ForEach(collider => collider.enabled = enabled);
-
     // Possesses the specified enemy
     internal void Possess(EnemyAI enemy) {
         this.Unpossess();
+        if (enemy.isEnemyDead) return;
```

Comment:
Is there a reason for putting `unpossess` before this? I think this would make the player unpossess if they tried to possess a dead enemy.

Reply:
You right, fixed.